content/en/about/logo.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Logo"
+draft: false
+---
+
+<img src="/images/logo-big.png" class="img-fluid logo-home pr-5" style="float: left;">
```

Comment:
I just copied this image insertion code from [the home page](https://github.com/InnerSourceCommons/innersourcecommons.org/blob/master/content/en/_index.md).  I don't know exactly what it all means, but it looks good and so left it.